context/app/static/js/components/entity-search/sidebar/ConfigureSearchTable/utils.js

```diff
@@ -1,9 +1,20 @@
-function getMetadataFieldsSortedByEntityTypeThenFieldName(fields) {
-  return Object.entries(fields)
-    .sort(([fieldNameA, entityTypeA], [fieldNameB, entityTypeB]) => {
-      return entityTypeA.localeCompare(entityTypeB) || fieldNameA.localeCompare(fieldNameB);
-    })
-    .map(([fieldName, fieldEntityType]) => ({ fieldName, fieldEntityType }));
+function getFieldEntriesSortedByConfigureGroup(fields) {
+  return Object.entries(fields).sort(
+    (
+      [, { configureGroup: configureGroupA, label: labelA }],
+      [, { configureGroup: configureGroupB, label: labelB }],
+    ) => {
+      // put 'General' configure group at top
```

Comment:
```suggestion
      // Sort 'General' configure group to top:
```
Capitalizing the first letter of comments, like I mentioned before. :)